### ISSUE TITLE
Is this the swiss leaks data?

### ISSUE THREAD

**Issue by navjotk** [swissleaks/swiss_leaks_data#1]
This data looks like the Panama Papers data. Is this really from the HSBC leak?

**Comment by michaelcoyote** [comment]
Yes.  Panama Papers wasn't released until April 3, 2016 and the data set is supposedly much larger.

This came from: https://www.icij.org/project/swiss-leaks
